code/src/components/Drink.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+export const Drink = ({ drink, setDrink }) => {
+  const handleDrinkChange = (event) => {
+    setDrink(event.target.value);
+  }
+  return (
+    <>
+      <div className="question">
+        <h2>Sounds delicious!
+            And drink?
+        </h2>
+      </div>
+      <div className="select-container">
+        <select
+          className="select"
+          id="drink"
+          value={drink}
+          onChange={handleDrinkChange}>
+          <option value="">Menu</option>
```

Comment:
I think you can add "disable" either before or after "value" for the MENU line and that will stop being an acceptable value. Then the user would have to pick something else from the dropdown.